tool/tctl/common/tctl.go

```diff
@@ -289,6 +290,9 @@ func tunnelAddr(webAddr utils.NetAddr, settings webclient.ProxySettings) (string
 		return settings.SSH.TunnelPublicAddr, nil
 	}
 
+	if settings.ALPNSNIListenerEnabled {
```

Comment:
This will break legacy client behaviour. Old client should connect to old reverse tunnel port and if proxy will always return ALPN SNI listener port for ReverseTunnel address the old `tctl` authDial behaviour won't work.